### PR TITLE
Add inline healing rolls to Vital Beacon

### LINKED ISSUE #16866: Vital Beacon inline healing
The inline healing roll in the vital beacon spell is rolling untyped healing instead of vitality healing.
image

### PR DESCRIPTION
**Pull request by pedrogrullada**
Closes #16866
I added inline healing rolls to the spell itself and removed the link to the effect, which is a purely tracking effect that should probably be discontinued. I added vitality to the rolls on the effect too though, for people who will still be using it.

### CODE REVIEW

**Comment by simonward** [comment]
> I added inline healing rolls to the spell itself and removed the link to the effect, which is a purely tracking effect that should probably be discontinued.

The effect has been handy for my games. I'll leave it up to @TikaelSol as to whether the link should be removed.

File: packs/spell-effects/spell-effect-vital-beacon.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
...
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Vital Beacon]</p>\n<p>Once per round, either you or an ally can use an Interact action to supplicate and lay hands upon you to regain Hit Points. It restores @Damage[(@item.level)d10[healing,vitality]|shortLabel] Hit Points to the first creature, @Damage[(@item.level)d8[healing,vitality]|shortLabel] Hit Points to the second, @Damage[(@item.level)d6[healing,vitality]|shortLabel] Hit Points to the third, and @Damage[(@item.level)d4[healing,vitality]|shortLabel] Hit Points to the fourth, after which the spell ends.</p>"
```
Comment (TikaelSol) [inline packs/spell-effects/spell-effect-vital-beacon.json:18]: We can maybe justify the effect by using a counter? 2 + 2*badge value gives the die size, can we use that in the formula to roll so there's just one formula and people can track it counting down from 4 to 1?

Comment (pedrogrullada) [reply packs/spell-effects/spell-effect-vital-beacon.json:18]: Wait, die size is resolvable?

Comment (TikaelSol) [reply packs/spell-effects/spell-effect-vital-beacon.json:18]: `@Damage` does a better job of resolving than other inlines. Even the damage type is resolvable.

Comment (TikaelSol) [reply packs/spell-effects/spell-effect-vital-beacon.json:18]: Though, it has to use `@` notation, not `{}` which is awkward but 🤷

Comment (pedrogrullada) [reply packs/spell-effects/spell-effect-vital-beacon.json:18]: I knew about the damage type but it never even occurred to me to try with die sizes. That's neat, I'll add that later then!

**Pull request merged**